go.mod

```diff
@@ -3,17 +3,28 @@ module github.com/hashicorp/vault-plugin-auth-kubernetes
 go 1.12
 
 require (
+	github.com/Masterminds/semver v1.4.2-0.20190429190951-059deebd1619 // indirect
```

Comment:
Would it be possible to remove the changes to `go.mod` and `go.sum` in this pull request? In general I believe we are moving to not including these in pull requests, and I don't see where any of these are either used here or required an update.